src/index.js

```diff
@@ -1 +1,86 @@
-console.log('%c HI', 'color: firebrick')
+const init = () => {
+  const dogImgUrl = "https://dog.ceo/api/breeds/image/random/4"
+  const breedUrl = "https://dog.ceo/api/breeds/list/all"
+  const dogBreedSection = document.getElementById("dog-breeds")
+  let dogBreedList
+
+  const displayDogImgs = (imgArray) => {
+    imgArray.forEach(dogImg => {
+      const imgSection = document.getElementById("dog-image-container")
+      const imgTag = document.createElement("img")
+      imgTag.src = dogImg
+      imgTag.style.height = "500px"
+      imgTag.style.width = "500px"
+      imgSection.appendChild(imgTag)
+    });
+  }
+
+  const changeColor = () => {
+    for (const listItem of dogBreedList){
+      listItem.addEventListener("click", () => {
+        if (listItem.style.color === "red"){
+          listItem.style.color = ""
+        } else {
+        listItem.style.color = "red"
+        }
+      })
+    }
+  }
+
+  const sortDogBreeds = () => {
+    const sortDropdown = document.getElementById("breed-dropdown") 
+    const originalList = dogBreedSection.innerHTML
+    sortDropdown.addEventListener("click", (event) => {
+      dogBreedSection.innerHTML = originalList
+      let userOption = event.target.value 
+      let sortedList = []
+      for (const listItem of dogBreedList){
+        if(listItem.textContent.charAt(0) === userOption){
+          sortedList.push(listItem)
+        }
+      }
```

Comment:
Which enumerable would be better suited for this application?

Reply:
I think dogbreed would be a better name for the enumerable. listItem doesn't really tell you anything

Reply:
Not the name, but which *enumerable*. Ex: `forEarch`, `map`, etc.

Reply:
Ah, I see now. Yeah, I noticed that yesterday when seeing your solution. If I did that, I could easily do the above with filter. Since filter will return back an array of all the elements that return true on the boolean expression I pass into it.